src/lib.rs

```diff
@@ -20,15 +22,18 @@ mod state;
 pub use dusk_abi;
 
 pub use call_context::StandardABI;
-pub use contract::Contract;
+pub use contract::{Contract, ContractId};
 pub use gas::{Gas, GasMeter};
 pub use state::NetworkState;
 
-#[derive(Debug, Fail)]
+#[derive(Fail)]
 /// The errors that can happen while executing the VM
-pub enum VMError {
-    /// An argument in `RuntimeArgs` is missing
-    MissingArgument,
+pub enum VMError<S>
```

Comment:
The proof verification failure don't seem to be contemplated in the error variants

Reply:
We will adapt the error type when we need it.